custom_components/localtuya/config_flow.py

```diff
@@ -289,8 +287,8 @@ def _convert_entity(conf):
         self.platform = user_input[CONF_PLATFORM]
 
         if len(user_input.get(CONF_SWITCHES, [])) > 0:
-            for switch_conf in user_input[CONF_SWITCHES].values():
-                self.entities.append(_convert_entity(switch_conf))
+                    for switch_conf in user_input[CONF_SWITCHES].values():
```

Comment:
This indentation doesn't look right. I think you can revert all the changes in this file as nothing should have to change here.

Reply:
Ok, i'll revert config_flow.py to current master